packages/fluentui/react-northstar/src/themes/teams-dark/siteVariables.ts

```diff
@@ -11,10 +11,10 @@ export const focusOuterBorderColor = colors.white;
 //
 // SHADOW LEVELS
 //
-export const shadowLevel1 = '0 .2rem .4rem -.075rem rgba(0, 0, 0, .25)';
-export const shadowLevel2 = '0 .4rem 0.7rem -0.1rem rgba(0, 0, 0, .25)';
-export const shadowLevel3 = '0 .8rem 1rem -0.2rem rgba(0, 0, 0, .25)';
-export const shadowLevel4 = '0 .6rem 1.8rem -0.4rem rgba(0, 0, 0, .25)';
+export const shadowLevel1 = '0 0 2px rgba(0,0,0,0.30), 0 2px 4px rgba(0,0,0,0.25)';
```

Comment:
aren't these changes for v2 theme? do we want to modify v1 theme as well?

Reply:
Thank you Juraj I am gonna ask this, in the task was not specified!